src/js/common/schemaform/SchemaForm.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import _ from 'lodash/fp';
+import Form from 'react-jsonschema-form';
+
+import { uiSchemaValidate, transformErrors } from './validation';
+import Address from './Address';
+import FieldTemplate from './FieldTemplate';
+import * as reviewWidgets from './review/widgets';
+import ReviewFieldTemplate from './review/ReviewFieldTemplate';
+import widgets from './widgets/index';
+import ObjectField from './ObjectField';
+import ArrayField from './ArrayField';
+import ReviewObjectField from './review/ObjectField';
+import { scrollToFirstError } from '../utils/helpers';
+
+const fields = {
+  ObjectField,
+  ArrayField,
+  address: Address
+};
+
+const reviewFields = {
+  ObjectField: ReviewObjectField,
+  ArrayField,
+  address: ReviewObjectField
+};
+
+/*
+ * Each page uses this component and passes in config. This is where most of the page level
+ * form logic should live.
+ */
+class SchemaForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.validate = this.validate.bind(this);
```

Comment:
Yeah, that's the one.

I don't know about the googlability of it, though; if I were to google it, I'd probably search for `javascript =>` which leads me right to the [MDN page on arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), and once it has a name, it's easy to find more about it.

That's my two cents anyhow. With ES6, it's becoming pretty standard.